app.py

```diff
@@ -8,13 +8,55 @@
     PicklePersistence,
 )
 from ptbcontrib.ptb_jobstores.sqlalchemy import PTBSQLAlchemyJobStore
+
+import grpc
+from opentelemetry import metrics
+from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import (
+    OTLPMetricExporter,
+)
+from opentelemetry.sdk import metrics as sdkmetrics
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import (
+    AggregationTemporality,
+    PeriodicExportingMetricReader,
+)
+from opentelemetry.sdk.resources import Resource
+
 from src.custom_filters import filter_bot_added
 from src.logging import tg_logger
 from src import handlers
 import os
+import time
+
+
+temporality_delta = {
+    sdkmetrics.Counter: AggregationTemporality.DELTA,
+    sdkmetrics.UpDownCounter: AggregationTemporality.DELTA,
+    sdkmetrics.Histogram: AggregationTemporality.DELTA,
+    sdkmetrics.ObservableCounter: AggregationTemporality.DELTA,
+    sdkmetrics.ObservableUpDownCounter: AggregationTemporality.DELTA,
+    sdkmetrics.ObservableGauge: AggregationTemporality.DELTA,
+}
 
 
 def main():
+    dsn = os.environ["UPTRACE_DSN"]
```

Comment:
better to use `os.environ.get()` -- currently it will throw if `UPTRACE_DSN` is undefined (which is likely to happen while developing)